src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -302,8 +310,7 @@ class InvoiceForm extends Component {
                       </Col>
                       <Col data-test-col-accounting-code xs={3}>
                         <FieldSelection
-                          dataOptions={[]}
-                          disabled
+                          dataOptions={getCodeOptions(accountCodes)}
                           label={<FormattedMessage id="ui-invoice.invoice.accountingCode" />}
                           name="accountingCode"
                           required
```

Comment:
it's not required field, AFAIK

Reply:
also `validate` property could be removed

Reply:
fixed